lib/tasks/populate.rake

```diff
@@ -128,8 +128,9 @@ namespace :spree_roles do
     desc "Create admin username and password"
     task populate: :environment do
       default_permission = make_permission('default-permissions', 0)
+      read_variant_permission = make_permission('can-read-spree/variants', 0)
```

Comment:
I would prefer adding additional permissions to the default set in the shop repo, like we did in the `lib/tasks/warehouse_role.rake`

Reply:
per your request - https://github.com/TerraCycleUS/tc-shop/pull/124